Dockerfile

```diff
@@ -1,17 +1,112 @@
+
+## Install Java 
+RUN apt-get update \
+    && apt-get install -t jessie-backports -y openjdk-8-jdk \
+    && echo "MAVEN IS NOT IN THE UPSTREAM LIST (JOSH)" \
+    && apt-get install -y maven \ 
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean 
 
-RUN install -o jovyan -g users -d /notebooks /opt/omero
+ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+RUN rm -rf /usr/lib/jvm/java
+RUN ln -s  /usr/lib/jvm/java-8-openjdk-amd64 /usr/lib/jvm/java
+RUN R CMD javareconf
+    
+## make sure Java can be found in rApache and other daemons not looking in R ldpaths
+RUN echo "/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/amd64/server/" > /etc/ld.so.conf.d/rJava.conf
+RUN /sbin/ldconfig
 
-USER jovyan
+###
+### Fix install2.r
+### --------------
+### See https://github.com/rocker-org/rocker/blob/e9758030e435915d5e6f21aaab0fc35a5a8efaae/r-base/Dockerfile#L41
+### https://github.com/rocker-org/rocker/issues/149
+### --------------
+## Now install R and littler, and create a link for littler in /usr/local/bin
+## Also set a default CRAN repo, and make sure littler knows about it too
+RUN apt-get update \
+        && echo "MIGRATE THIS BLOCK TO THE APT-GET ABOVE" \
+	&& apt-get install -y --no-install-recommends \
+		littler \
+        && echo 'options(repos = c(CRAN = "https://cran.rstudio.com/"), download.file.method = "libcurl")' >> /etc/R/Rprofile.site \
+        && echo 'source("/etc/R/Rprofile.site")' >> /etc/littler.r \
+	&& ln -s /usr/share/doc/littler/examples/install.r /usr/local/bin/install.r \
+	&& ln -s /usr/share/doc/littler/examples/install2.r /usr/local/bin/install2.r \
+	&& ln -s /usr/share/doc/littler/examples/installGithub.r /usr/local/bin/installGithub.r \
+	&& ln -s /usr/share/doc/littler/examples/testInstalled.r /usr/local/bin/testInstalled.r \
+	&& install.r docopt \
+	&& rm -rf /tmp/downloaded_packages/ /tmp/*.rds \
+	&& rm -rf /var/lib/apt/lists/*
+
+## Install rJava package
+ENV PATH=$PATH:/usr/lib/jvm/java-8-openjdk-amd64/bin
+RUN conda install --quiet --yes -c r r-rjava=0.9_8
+## TODO: move me to other environment variables
+## RUN install2.r --error rJava \
+##  && rm -rf /tmp/downloaded_packages/ /tmp/*.rds
+
+# Changed from rOMERO-gateway/Dockerfile
+RUN chown omero /usr/local/lib/R/site-library
+##
+##
+#########################################################################################
+
+
+# install romero
+RUN mkdir /romero \
+ && wget https://raw.githubusercontent.com/ome/rOMERO-gateway/master/install.R \
+ && Rscript install.R --user=snoopycrimecop --branch=master/merge/daily
+
+RUN install -o omero -g users -d /notebooks /opt/omero
+
+USER omero
 
 RUN pip2 install omego && \
     cd /opt/omero && \
-    /opt/conda/envs/python2/bin/omego download --ice 3.6 server --release 5.2 --sym OMERO.server && \
+    /opt/conda/envs/python2/bin/omego download --ice 3.6 server --release 5.3 --sym OMERO.server && \
     rm -f OMERO.server-*.zip && \
     echo /opt/omero/OMERO.server/lib/python > \
```

Comment:
Question for you: Standardise on `/opt/omero/server/OMERO.server` as used in images deployed with ansible, or keep this adhoc path since it's not a running server?